lean4-infoview/src/infoview/main.tsx

```diff
@@ -21,6 +21,14 @@ function Main(props: {}) {
     if (!props) { return null }
     const ec = React.useContext(EditorContext);
 
+    const [htmlBlob, setHtmlBlob] = React.useState<string>('')
```

Comment:
The changes to the infoview are no longer necessary now.

Reply:
Fixed